ferrocene/tools/self-test/src/test_utils.rs

```diff
@@ -233,15 +233,10 @@ impl Default for CliVersionContent<'_> {
 
 impl CliVersionContent<'_> {
     pub(crate) fn serialize(&self) -> String {
-        let mut buffer = String::new();
-        buffer.push_str("release: ");
-        buffer.push_str(self.release);
-        buffer.push_str("\nhost: ");
-        buffer.push_str(self.host);
-        buffer.push_str("\ncommit-hash: ");
-        buffer.push_str(self.commit_hash);
-        buffer.push('\n');
-        buffer
+        format!(
+            "release: {}\nhost: {}\ncommit-hash: {}\n",
+            self.release, self.host, self.commit_hash
```

Comment:
would be nice to get this looking close to how it will look when executed (like was done in [parent commit](https://github.com/ferrocene/ferrocene/pull/392/commits/feb1dd1a897252d39fb45df7a1072cc3e4023d80#diff-76f740a27d65cacdf8abe8a57b7ede0e21ea3482062d08c992aa787b2298b5b1R129-R132))

Reply:
Changed it.